CHANGELOG.md

```diff
@@ -0,0 +1,80 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+## [v0.7.0](https://github.com/voxpupuli/hiera-eyaml-gpg/tree/v0.7.0) (2019-04-24)
+
+[Full Changelog](https://github.com/voxpupuli/hiera-eyaml-gpg/compare/vp_migration...v0.7.0)
+
+This is the first release of `hiera-eyaml-gpg` since the project was migrated to [Vox Pupuli](https://voxpupuli.org/).  We're pleased to announce that this project should now work with Puppet 6 (jruby 9k puppetserver).  Special thanks to [seanmil](https://github.com/seanmil) for his work on this.
```

Comment:
This comes from the release summary ticket. https://github.com/voxpupuli/hiera-eyaml-gpg/issues/59